Sources/ProjectDescription/Version.swift

```diff
@@ -105,11 +105,11 @@ extension Version {
             .split(separator: ".", maxSplits: 2, omittingEmptySubsequences: false)
             .map(String.init).compactMap { Int($0) }.filter { $0 >= 0 }
 
-        guard requiredComponents.count == 3 else { return nil }
+        guard requiredComponents.count > 0 else { return nil }
 
         major = requiredComponents[0]
-        minor = requiredComponents[1]
-        patch = requiredComponents[2]
+        minor = requiredComponents.count > 1 ? requiredComponents[1] : 0
+        patch = requiredComponents.count > 2 ? requiredComponents[2] : 0
```

Comment:
I think it better reflects the logic:
```suggestion
        minor = requiredComponents.count >= 2 ? requiredComponents[1] : 0
        patch = requiredComponents.count >= 3 ? requiredComponents[2] : 0
```